File: patterns/c-patterns/pattern115.c
// Pattern 115: ...

// **
// **
// ****
// ****
// ******
// ******

#include <stdio.h>

int main()
{
    int i, j, c, k;

    for (i = 1; i <= 6; i++)
    {
        k = (i % 2 != 0) ? i + 1 : i;

        for (j = 0; j < k; j++)
        {
            printf("*");
        }
        printf("\n");
    }

    return 0;
}

File: patterns/javascript-patterns/pattern115.js
// Pattern 115: ...

// **
// **
// ****
// ****
// ******
// ******

for (let i = 1; i <= 6; i++) {
    let k = (i % 2 !== 0) ? i + 1 : i;

    let line = '';
    for (let j = 0; j < k; j++) {
        line += '*';
    }

    console.log(line);
}
